frontend/src/pages/LearningCourse/FormDialog/FormDialog.tsx

```diff
@@ -37,9 +34,7 @@ const FormDialog: React.FC<IFormDialog> = ({ dialogOpen, handleDialogClose }) =>
             <CloseIcon alt="close" src={close} />
           </CloseButton>
         </CloseButtonBox>
-        <StyledDialogTitle>
-          <CountDownTimer duration={TEST_TIME_REMAINS_IN_SECONDS} />
-        </StyledDialogTitle>
+        <StyledDialogTitle>2 h 48 min 48 sec left</StyledDialogTitle>
```

Comment:
Do we really need to save mocked data?

Reply:
I'll implement a helper which will convert data from server from seconds to time in hours minutes and seconds in V1-073. Now it was changed because of QA request (from timer to static mocked data)